docs/usage.rst

```diff
@@ -39,24 +39,16 @@ You can run every session by just executing `nox` without any arguments:
 
     nox
 
-The order that sessions are executed is determined by how they are defined in
-your noxfile. if you use the :func:`decorator <nox.session>`, ``@nox.session``,
-then sessions will be run by nox in the order that they appear in the noxfile.
-If you define sessions using the naming convention, ``def session_<name>```
-they run in alphabetical order.
-
-If you mix and match the two methods, all sessions defined using the decorator
-are run first (in order), followed by all sessions defined by the naming
-convention, alphabetically.
+The order that sessions are executed is the order that they appear in the Noxfile.
 
 Specifying one or more sessions
 -------------------------------
 
 By default nox will run all sessions defined in the noxfile. However, you can choose to run a particular set of them using ``--session``, ``-s``, or ``-e``::
 
-    nox --session py27
-    nox -s lint py27
-    nox -e py34
+    nox --session tests
+    nox -s lint tests
+    nox -e lint
 
 Nox will run these sessions in the same order they are specified.
```

Comment:
I would really like a `-p/--python` argument to limit to sessions for a particular interpreter.

Example:
`nox -s tests -p 3.7`

Reply:
(Although it would be fair to do it in a follow-up, or tell me to do so. :-p)

Reply:
Let's do it in a separate PR https://github.com/theacodes/nox/issues/95

Reply:
Ack.